package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "visma-e-commerce",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "dependencies": {
+    "array-sort": "^1.0.0",
+    "call-bind": "^1.0.2",
+    "error-ex": "^1.3.2",
+    "for-own": "^1.0.0",
+    "fs.realpath": "^1.0.0",
+    "function-bind": "^1.1.1",
+    "get-intrinsic": "^1.1.1",
+    "get-caller-file": "^1.0.3",
+    "get-value": "^2.0.6",
+    "graceful-fs": "^4.2.6",
+    "gulp-autoprefixer": "^7.0.1",
+    "gulp": "^4.0.2",
```

Comment:
Wooooow. Are all of those modules really necessary for gulp? Do they really need to be listed explicitly? I reckon that gulp itself uses them and they should probably be picked as dependencies of gulp and would be handled by npm.  
You should only need to list dependencies you are really using and peer dependencies of those if there are any.